### ISSUE TITLE
Recommend using serial key when online activation fails

### ISSUE THREAD

**Issue by nbolton** [synergy/synergy#5020]
Problem: Online activation (email/password) can fail for a number of reasons, but often the user is left wondering why.

Solution: When online activation fails, recommend the user tries offline activation (serial key).

**Comment by dhiva** [comment]
The activation was successful for me in Windows 7 but the application still shows "(unregistered)".

**Comment by aj4mq** [comment]
Appears to be fixed in 1.8.1-beta-f7377d6, both 32- and 64-bit windows versions tested.

Also note that the serial number is available for entry on the first screen.

**Comment by kdoteu** [comment]
Working for me

![image](https://cloud.githubusercontent.com/assets/3602457/16535259/52762ce8-3fe6-11e6-87b9-d91053fdb193.png)

Version: synergy-v1.8.1-beta-f7377d6
OS: Windows 7 64 Bit

**Comment by SpaceRangerWes** [comment]
![capture](https://cloud.githubusercontent.com/assets/4719435/16536447/47828fc2-3fb7-11e6-8b65-ab505851836c.PNG)

Still an error/issue on Windows 10 Enterprise 64 bit.

**Comment by mikeo-bixly** [comment]
Worked as expected.
OS: Linux (Fedora 24)
Version: v1.8.1-beta-f7377d6-Linux-x86_64

**Comment by chaoservices** [comment]
A-OK. Win 7 x64

**Comment by Imaginable** [comment]
Working as expected.

OS: Windows 10 64bit
Version: v1.8.1-beta-f7377d6

OS: OS X El Capitan V10.11.5
Version: v1.8.2-beta-f7377d6

**Comment by Kays** [comment]
Working as expected.
OS: Debian Jessie (8.5)
Synergy v1.8.1-beta-f7377d6

![2016-07-01-20 23 03_556x465](https://cloud.githubusercontent.com/assets/505500/16537179/8ed97d3c-3fca-11e6-897b-6c346327035f.png)

**Comment by Silarn** [comment]
I can verify the error posted by @SpaceRangerWes. But the serial suggestion does still work.

**Comment by Verequies** [comment]
I can also verify the error posted by @SpaceRangerWes on Windows 10 Enterprise 64-bit. And that serial suggestion still works.
![issue 5020](https://cloud.githubusercontent.com/assets/282071/16537765/7416c6f0-404f-11e6-9a70-063e4ecde724.png)

**Comment by yidhoe** [comment]
Works for me:
OS: Windows 64bit (Clients & Server)
Version: synergy-v1.8.1-beta-f7377d6

Valid timed Key: ns.dll is loaded

**Comment by Zygislovas** [comment]
There are some cases when it works and when it does not.
**SERVER** system info:
Windows 10 Enterprise x64
**CLIENT** system info:
Windows 8.1 Pro x64

**Version**: synergy-v1.8.1-beta-f7377d6

For e.g.: If I use probably non existing user email as test@test.com and any random password - it notifies me that login failed and once I close that message it asks if I want to use key instead.
However, if I use existing user(I've used my own that is registered in Synergy, but it does not have any valid subscription) - then wizard will not prompt me to use key...

I'm not sure if it should ask every single time login fails, but currently it asks only once while you stay on the same ACTIVATE wizard step. I mean if you go back to WELCOME step and then go Next and try to fail login again, it will suggest you to enter key instead.

**Comment by MartinNuc** [comment]
OS X 10.11.5

when I turn off internet connection:
![image](https://cloud.githubusercontent.com/assets/681176/16542459/6955fd1c-40a9-11e6-9143-04dd998532e6.png)

after clicking on OK it just disappears. However after I restored internet connection clicking on OK resulted in prompt if I want to use serial key.

**Comment by cletsimon** [comment]
OS X 10.10.5
Activation setup not showing after reinstallation
![screen shot 2016-07-03 at 6 49 50 pm](https://cloud.githubusercontent.com/assets/5573468/16545027/3da1bdcc-414f-11e6-8ea7-fcbb034a3814.png)
![screen shot 2016-07-03 at 6 50 44 pm](https://cloud.githubusercontent.com/assets/5573468/16545026/3da19414-414f-11e6-8577-e71a5cad085a.png)

**Comment by cletsimon** [comment]
looks good OSX 10.10.5 after 3rd try
![screen shot 2016-07-03 at 7 18 56 pm](https://cloud.githubusercontent.com/assets/5573468/16545127/279856b8-4153-11e6-8d70-4a8bbe538255.png)

**Comment by GollyRojer** [comment]
Windows 10 Pro
synergy-v1.8.1-beta-f7377d6
Works as expected.  I do see the "FATAL" errors as reported by @SpaceRangerWes, but they are not fatal in that the dialog window remains functional.  On third attempt the messagebox offering the serial key appears.
As noted by @aj4mq, the serial key option is available from the first screen.

**Comment by ruicovelo** [comment]
On Mac OS X 10.11.5.

I get the prompt as expected. I then tried using the serial key from the Issue #4715 test. After the plugins are installed, synergy closes. I know this was not part of this test but might this be a problem?
